js/js.go

```diff
@@ -73,7 +73,7 @@ func (o *Object) Uint64() uint64 { return o.object.Uint64() }
 // Float returns the object converted to float64 according to JavaScript type conversions (parseFloat).
 func (o *Object) Float() float64 { return o.object.Float() }
 
-// Interface returns the object converted to interface{}. See GopherJS' README for details.
+// Interface returns the object converted to interface{}. See table in package comment for details.
```

Comment:
It's been a few days, and I still haven't heard back on my comment above. Since this has approval, I'll assume it's convincing and merge this in order to make progress.

If there's anything that can be improved or needs to be followed up, another PR can be made.